res/translations/mixxx.ts

```diff
@@ -9499,6 +9504,61 @@ Fully right: end of the effect period</source>
         <translation></translation>
     </message>
 </context>
+<context>
+    <name>RekordboxFeature</name>
+    <message>
+        <location filename="../../src/library/rekordbox/rekordboxfeature.cpp" line="1048"/>
+        <location filename="../../src/library/rekordbox/rekordboxfeature.cpp" line="1111"/>
+        <location filename="../../src/library/rekordbox/rekordboxfeature.cpp" line="1265"/>
+        <source>Rekordbox</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../src/library/rekordbox/rekordboxfeature.cpp" line="1112"/>
+        <source>Reads the following from Rekordbox prepared removable devices:</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../src/library/rekordbox/rekordboxfeature.cpp" line="1116"/>
+        <source>Playlists</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../src/library/rekordbox/rekordboxfeature.cpp" line="1117"/>
+        <source>Folders</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../src/library/rekordbox/rekordboxfeature.cpp" line="1118"/>
+        <source>First memory cue</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../src/library/rekordbox/rekordboxfeature.cpp" line="1119"/>
+        <source>First memory loop</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../src/library/rekordbox/rekordboxfeature.cpp" line="1120"/>
+        <source>Hot cues</source>
```

Comment:
The next occurrence `Hot Cue` ≠ `Hotcue`.
How we want to proceed? We can treat this as proper name, since Pioneer decided for this spelling in Rekordbox, but I´d rather not change all affected strings in app, manual, website.

Reply:
I think users can understand both. So it is not a usability issue to start now using "Hot Cue" where we stumble over it and keep other occurences unchanged.

Reply:
Remember that, last minute string changed leads to untranslated strings. That IS a strong usability issue for non English speaking users.
So if we decide for a mass change, postpone it to 2.4.

Reply:
> Remember that, last minute string changed leads to untranslated strings. That IS a strong usability issue for non English speaking users.
> So if we decide for a mass change, postpone it to 2.4.

I don't know what you are referring to, there was no intention to mass change anything.

Nonetheless i do not think it is a good idea to allow into different spelling for a term, since translators can currently rely on our [glossary of terms](https://www.transifex.com/mixxx-dj-software/mixxxdj-manual/glossary/en/). In previous versions, certain efforts have been made to keep everything in sync.

Transifex performs a series of translation checks before accepting any new translation, one of them matches against glossary terms. It will throw a warning.

Reply:
This sounds reasonable. What do we use in German by the way?
Hot-Cue?

Reply:
We use [Hotcue](https://www.transifex.com/mixxx-dj-software/mixxxdj/viewstrings/#de/mixxx2-3/184527064?q=text%3Ahotcue)